fizzbuzz/fizzbuzz.c

```diff
@@ -10,7 +10,31 @@
 */
 int fizzbuzz(int n)
 {
+    int counter = 0;
+    int i = 1;
 
+    while (i <= n)
```

Comment:
be careful of how you would go about a while loop, normally when you would want to have this sort loop make sure theres some sort of closure :)